Test/CoreSDK.Test/Shared/Extensibility/Implementation/Tracing/DiagnosticsTelemetryModuleTest.cs

```diff
@@ -53,20 +56,47 @@ public void TestDiagnosticsModuleSetSeverity()
             {
                 initializedModule.Initialize(new TelemetryConfiguration());
                 
-                Assert.AreEqual(EventLevel.Error.ToString(), initializedModule.Severity);
+                Assert.Equal(EventLevel.Error.ToString(), initializedModule.Severity);
 
                 initializedModule.Severity = "Informational";
 
-                Assert.AreEqual(EventLevel.Informational, initializedModule.EventListener.LogLevel);
+                Assert.Equal(EventLevel.Informational, initializedModule.EventListener.LogLevel);
             }
         }
 
         [TestMethod]
         public void TestDiagnosticModuleDoesNotThrowIfInitailizedTwice()
         {
-            DiagnosticsTelemetryModule module = new DiagnosticsTelemetryModule();
-            module.Initialize(TelemetryConfiguration.Active);
-            module.Initialize(TelemetryConfiguration.Active);
+            using (DiagnosticsTelemetryModule module = new DiagnosticsTelemetryModule())
+            {
+                module.Initialize(new TelemetryConfiguration());
+                module.Initialize(new TelemetryConfiguration());
```

Comment:
Because test checks that Initialize would not throw if called twice. In the test I changed only TC.Active on empty configuration. It is better since recently we started adding same module by defaut in TC.Active